packages/react/popover/src/Popover.tsx

```diff
@@ -246,69 +314,31 @@ const PopoverContentImpl = React.forwardRef((props, forwardedRef) => {
   // the last element in the DOM (beacuse of the `Portal`)
   useFocusGuards();
 
-  // Hide everything from ARIA except the content
-  React.useEffect(() => {
-    const content = contentRef.current;
-    if (content) return hideOthers(content);
-  }, []);
-
   return (
     <PortalWrapper>
       <ScrollLockWrapper>
```

Comment:
The outside scrolling should be part of modality, so it shouldn't be in the `Impl` anymore, but rather only in the modal version.

Reply:
Good point, was wondering about this.